components/ws-manager-bridge/leeway.Dockerfile

```diff
@@ -2,13 +2,13 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License.AGPL.txt in the project root for license information.
 
-FROM node:18.17.1-slim as builder
+FROM node:20-slim as builder
 COPY components-ws-manager-bridge--app /installer/
 
 WORKDIR /app
 RUN /installer/install.sh
 
-FROM cgr.dev/chainguard/node:18.17.1@sha256:af073516c203b6bd0b55a77a806a0950b486f2e9ea7387a32b0f41ea72f20886
+FROM cgr.dev/chainguard/node:20@sha256:f30d39c6980f0a50119f2aa269498307a80c2654928d8e23bb25431b9cbbdc4f
```

Comment:
This we should do for sure! Did not see it at first due to the log changes.
I would do a quick test in a preview. :+1:

Reply:
Unless we have a concrete reason, I would not want to update node. Version 18 is supported until next year April.
Also if we do the update we should do it for all our node servers.